### PR TITLE
clean up unused code

### PR DESCRIPTION
**Pull request by hamelsmu**
Get rid of unused modules:

1. read.py
2. mkdocs.py

Gets rid of unused imports, code and functions.  I tried my best to annotate where possible

cc: @jph00 @seeM

### CODE REVIEW

File: nbdev/release.py [hunk 1]
```diff
@@ -161,7 +159,6 @@ def pypi_details(name):
     "Version, URL, and SHA256 for `name` from pypi"
     ver = str(latest_pypi(name))
     pypi = pypi_json(f'{name}/{ver}')
-    info = pypi['info']
```
Comment (hamelsmu) [inline nbdev/release.py:164]: Was this line of intended to do something but forgotten?

File: nbdev/serve.py [hunk 2]
```diff
@@ -32,11 +32,6 @@ def _is_qpy(path:Path):
         vl = v.splitlines()
         if vl[0]=='---' and vl[-1]=='---': return '\n'.join(vl[1:-1])
 
-def _exec_py(fname):
-    "Exec a python script and warn on error"
-    try: subprocess.check_output('python ' + fname, shell=True)
-    except subprocess.CalledProcessError as cpe: warn(str(cpe))
-
```
Comment (hamelsmu) [inline nbdev/serve.py:39]: I couldn't find anywhere that this function was used

File: nbdev/sync.py [hunk 3]
```diff
@@ -51,7 +51,6 @@ def _update_lib(nbname, nb_locs, lib_name=None):
 # %% ../nbs/api/sync.ipynb 10
 @functools.lru_cache(maxsize=None)
 def _mod_files():
-    mdir = get_config().path('lib_path').parent
```
Comment (hamelsmu) [inline nbdev/sync.py:54]: `mdir` doesn't appear to be used

File: nbs/api/showdoc.ipynb [hunk 4]
```diff
@@ -53,6 +51,7 @@
    "outputs": [],
    "source": [
     "#|hide\n",
+    "from inspect import Parameter\n",
```
Comment (hamelsmu) [inline nbs/api/showdoc.ipynb:54]: `Parameter` only used for testing AFAICT so moved it to a non-exported cell

File: nbdev/release.py (hunk 1 above)
Comment (jph00) [reply nbdev/release.py:164]: Dunno

**Comment by jph00** [comment]
Very nice clean up

**Pull request merged**